src/test/java/fr/uge/dummychargepoint/socket/ConfigurationServerIntegrationTest.java

```diff
@@ -0,0 +1,63 @@
+package fr.uge.dummychargepoint.socket;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ConfigurationServerIntegrationTest {
+
+  private static final int RANDOM_PORT = 0;
+
+  private static ConfigurationServer server;
+
+  @BeforeAll
+  static void start() throws InterruptedException {
+    server = new ConfigurationServer(new InetSocketAddress(RANDOM_PORT));
+    server.start();
+    Thread.sleep(500);
+  }
+
+  @AfterAll
+  static void stop() throws InterruptedException {
+    server.stop();
+    Thread.sleep(500);
```

Comment:
why ?

Reply:
Since it gracefully stop the server, I think we should wait a little bit before being sure that the server is closed

Reply:
You're right, after changing with the annotation, we should not wait, [cba284c](https://github.com/LastProject-ESIEE/dummy-chargepoint/pull/6/commits/cba284c90fca7434c054e68ff645dfb64444a1a9)